runtime/testy/runtime.py

```diff
@@ -30,7 +30,10 @@
 
 allProcesses = {}
 
-def runtime():
+def runtime(testName=""):
+  testMode = testName != ""
+  maxIter = 3 if testMode else 0
```

Comment:
Again unrelated to the current changeset, but `maxIter` of 0 meaning unlimited is highly misleading. While this is standard practice in C due to the limits of the type system, for Python you may as well use `None` to make it clear that this is a signalling value.